### ISSUE TITLE
Cannot build any project from template due to "Could not copy the file Microsoft.WindowsAppSDK.Bootstrap.dll"

### ISSUE THREAD

**Issue by IsmailHassani** [microsoft/WindowsAppSDK#1777]
### Describe the bug

Any project I create that uses MS WindowsAppSDK fails on build ith this message.
```
 Build started...
1>------ Build started: Project: ClassLibrary4, Configuration: Debug Any CPU ------
1>  ClassLibrary4 -> C:\Users\x\source\repos\ClassLibrary4\bin\Debug\ClassLibrary4.dll
1>C:\Users\x\.nuget\packages\microsoft.windowsappsdk.foundation\1.0.0-experimental1\build\Microsoft.WindowsAppSDK.Bootstrap.targets(11,5): error MSB3030: Could not copy the file "C:\Users\x\.nuget\packages\microsoft.windowsappsdk.foundation\1.0.0-experimental1\build\..\runtimes\lib\native\AnyCPU\Microsoft.WindowsAppSDK.Bootstrap.dll" because it was not found.
========== Build: 0 succeeded, 1 failed, 0 up-to-date, 0 skipped ==========
```

### Steps to reproduce the bug

Create a new project:
 - Experimental Class Library (WInUI 3 in Desktop)
 - Class Library (WinUI 3 in Desktop)

And try to build it. 
It fails directly. Even if clean all obj/bin folders and clean the nuget cache.

### Expected behavior

It should build correctly.

### Screenshots

![image](https://user-images.githubusercontent.com/7172716/141377523-247dd438-0bd2-4ee0-ba9a-2f2c07a2e595.png)

### NuGet package version

1.0.0-preview3

### Packaging type

Unpackaged

### Windows version

Windows 11 (22000)

### IDE

Visual Studio 2022

### Additional context

Project is a class library from where i want to create a nuget package.

**Comment by pmpurifoy** [comment]
I believe building with AnyCPU is not supported. It is only x86, x64, and arm64.

**Comment by IsmailHassani** [comment]
Ok, maybe i try to do something wrong, but essentially i want to create 1 multi platform nuget from 1 project.

This is how the project file is set up:
It's automatically done by the template... so what am i missing here?

```
<Project Sdk="Microsoft.NET.Sdk">
  <PropertyGroup>
    <TargetFramework>net5.0-windows10.0.18362.0</TargetFramework>
    <TargetPlatformMinVersion>10.0.17763.0</TargetPlatformMinVersion>
    <RootNamespace>ClassLibrary3</RootNamespace>
    <RuntimeIdentifiers>win10-x86;win10-x64;win10-arm64</RuntimeIdentifiers>
    <UseWinUI>true</UseWinUI>
  </PropertyGroup>

  <ItemGroup>
      <PackageReference Include="Microsoft.WindowsAppSDK" Version="1.0.0-experimental1" />
      <PackageReference Include="Microsoft.WindowsAppSDK.Foundation" Version="1.0.0-experimental1" />
      <PackageReference Include="Microsoft.WindowsAppSDK.WinUI" Version="1.0.0-experimental1" />
      <PackageReference Include="Microsoft.WindowsAppSDK.InteractiveExperiences" Version="1.0.0-experimental1" />
  </ItemGroup>
</Project>
```

**Comment by ParkhutRoman** [comment]
@IsmailHassani 1.0.0-experimental1 != 1.0.0-preview3 . Wrong template

**Comment by IsmailHassani** [comment]
I have

> @IsmailHassani 1.0.0-experimental1 != 1.0.0-preview3 . Wrong template

I have this issue also with the preview3 template.

**Comment by ParkhutRoman** [comment]
> 
> 
> @ParkhutRoman I have this issue also with the preview3 template.

Windows 11 (22000) is way too old

**Comment by AdamBraden** [comment]
For .NET dlls, this is fixed in the 1.0 GA release - please update to the latest 1.0 nuget.

For .NET Exes, it is still broken, see #893
